Tests/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.8-slim-buster
+
+WORKDIR /scs-compliance
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+ADD . .
+
+ENTRYPOINT ./scs-compliance-check.py scs-compatible.yaml iaas
```

Comment:
Split this into `ENTRYPOINT` and `CMD`:

```
ENTRYPOINT ["./scs-compliance-check.py"]
CMD ["scs-compatible.yaml", "iaas"]

Reply:
You're right. I'm very new into building Docker images. The `CMD` should be adjustable by the user upon invoking the container. Let me check on how to achieve this.

Reply:
Should we replace the hard-coded YAML spec file and the layer with environment variables?

Reply:
The `CMD` trick was really helpful. Thank you @tibeer :pray: